core/src/client.rs

```diff
@@ -345,7 +345,9 @@ impl APIClient {
     }
 
     async fn make_headers(&self) -> Result<HeaderMap> {
+        let sdk_info: String = get_sdk_info();
         let mut headers = HeaderMap::new();
+        headers.insert("User-Agent", sdk_info.parse()?);
```

Comment:
We could set user agent with client builder, but not with every request.
ref: https://docs.rs/reqwest/latest/reqwest/struct.ClientBuilder.html#example